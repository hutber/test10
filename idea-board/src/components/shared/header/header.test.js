import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Header from './index';

const ShallowHeader = shallow(<Header copy='Idea Board' />);
describe('<Header />', () => {
	it('renders without crashing', () => {
		ShallowHeader
	});
	it('We correctly output copy', () => {

	});
	it('Creates a snapshot', () => {
		const HeaderInJson = renderer
			.create(<Header copy='Idea Board' />).toJSON();
		expect(HeaderInJson).toMatchSnapshot();
	})
})
