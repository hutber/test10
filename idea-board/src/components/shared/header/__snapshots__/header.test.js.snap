// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Creates a snapshot 1`] = `
<h1
  className="MuiTypography-root-1 MuiTypography-h1-13 MuiTypography-gutterBottom-28 MuiTypography-alignCenter-24"
>
  Idea Board
</h1>
`;
